addons/fabric8/fabric8.addon

```diff
@@ -0,0 +1,40 @@
+# Name: fabric8
+# Description: Installs fabric8 locally on Minishift
+
+# TODO enable CORS!
+
+oc login -u system:admin
```

Comment:
Ah yeah - lemme split that off as a separate issue: https://github.com/minishift/minishift/issues/1053